### PR TITLE
Add examples for cloud-config

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add examples for cloud-config.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: docs/cloud-config.md [hunk 1]
```diff
@@ -16,6 +16,40 @@ subnet-id=
...
+cloud=myhuaweicloud.com
+auth-url=https://iam.myhuaweicloud.com:443/v3
+
+[Vpc]
+id=1jk3u4ic4******02361dde2
+subnet-id=1183fe******bfdfc10d
+security-group-id=1c308fe5******be519e76f02
+```
+
+### Examples for Orange Cloud
```
Comment (niuzhenguo) [inline docs/cloud-config.md:36]: better to use Flexible Engine instead of Orange Cloud

Comment (chengxiangdong) [reply docs/cloud-config.md]: done

**Pull request merged**